doctr/travis.py

```diff
@@ -146,32 +146,20 @@ def setup_GitHub_push(deploy_repo, auth_type='deploy_key', full_key_path='github
     For ``auth_type='deploy_key'``, this sets up the remote with ssh access.
     """
 
-    if branch_whitelist is None:
+    if not branch_whitelist:
         branch_whitelist={'master'}
 
     if require_master is not None:
```

Comment:
But maybe we can do that in a different PR?  I'd like to get this in before people who have `pip install doctr` start experiencing pain